### PR TITLE
[docs]: Fix documentation for the Date Picker Component

### PR DESCRIPTION
**Pull request by Swapnil-2502**
This PR is for #11177

Summary: 

The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/datepicker

Content updates:

In the Event Table, update the "Description" for the given event [reference](https://docs.tooljet.com/docs/widgets/text-input#events) ( Please just check the sentence structure from the given reference )
In Exposed variable Table, update the "Description" and "How To Access" for the given action [reference](https://docs.tooljet.com/docs/widgets/text-input#exposed-variables) ( Please just check the sentence structure from the given reference )
Layout Table
Update the title name from "Layout" to "Devices" [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
Update the name of the first column from "Layout" to "Property"
Update the content in the "Description" and "Expected value" columns as per the given [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices) ( Please just check the sentence structure from the given reference )
Add a full stop at the end of the sentence in the "Description" and "Expected value" columns.

The changes are to be implemented in two files -
Next - ToolJet/docs/docs/widgets/datepicker.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/datepicker.md

Note: Please review this PR and let me know if further changes are needed. Thank you.

### CODE REVIEW

File: docs/docs/widgets/datepicker.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 <div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
 | Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/datepicker.md:96]: Hey, here you need to update the content of Description and Expected Value column by referring to the given [reference ](https://docs.tooljet.com/docs/widgets/text-input/#devices)in the issue.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @Swapnil-2502, please make the mentioned changes in both versions, thanks!

File: docs/docs/widgets/datepicker.md (hunk 1 above)
Comment (Swapnil-2502) [reply docs/docs/widgets/datepicker.md:96]: Okay let me check it again.

**Comment by RudraPratik30** [comment]
Hey @Swapnil-2502, can you please let us know if you are still working on this PR? Or if you are facing any issue then feel free to drop your query in our [Slack community](https://join.slack.com/t/tooljet/shared_invite/zt-2rk4w42t0-ZV_KJcWU9VL1BBEjnSHLCA).

**Comment by Swapnil-2502** [comment]
Yes, I am working on it. I will ask in case of any query.

**Comment by Swapnil-2502** [comment]
@RudraPratik30 

This is what expected 
Screenshot 2024-11-07 at 3 01 56 PM

And this is the [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
Screenshot 2024-11-07 at 3 02 15 PM

This is what I think needs to be done please verify and this will be exactly similar to the given reference, after which I will commit the changes in both the files.

**Comment by RudraPratik30** [comment]
Hey @Swapnil-2502,
Yes the screenshot looks correct to me, other than this you have to make some changes in the event table as well, as mentioned in the issue, make sure to push all the changes in both versions in a single PR, thanks again!

**Comment by Swapnil-2502** [comment]
Hi @RudraPratik30, I have updated the changes. If there are still changes to be made, I will be sorry for it. Please have a look and let me know.

**Review by RudraPratik30 — APPROVED** [review]
Hey @Swapnil-2502, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.
Your PR will be merged soon, thanks again!

**Comment by Swapnil-2502** [comment]
> Hey @Swapnil-2502, thanks for contributing! The changes look good to me and forwarding it to the team for final review. Your PR will be merged soon, thanks again!

Thank you for your time.

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Review by johnsoncherian — APPROVED** [review]

**Pull request merged**